evals/000-fundamentals/008-helper_fns/answer/bun.lock

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
one thing that's weird is that I have `bun.lockb` on my system but you're getting `bun.lock`? maybe we should just gitignore both of them...

Reply:
The lock file should be included in the repo

I suspect the reason for the two is the latest verison of bun (1.2) that was released a few days ago and the update introduced a text version of the lock file instead of the binary. 

https://bun.sh/blog/bun-v1.2

There is probably going to be a period where some people have the older version of some have the newer version of bun.

Reply:
ah cool. we can also just require >= 1.2 if that makes our lives easier